cluster/engine.go

```diff
@@ -347,6 +347,15 @@ func (e *Engine) TotalCpus() int64 {
 	return e.Cpus + (e.Cpus * e.overcommitRatio / 100)
 }
 
+// NodeLabels returns the assigned and standard labels
+func (e *Engine) NodeLabels() string {
+	labels := []string{}
```

Comment:
@noxiouz I initially did that and while updating the code changed it.. Lemme make that. Thanks for your suggestion